pkgs/development/python-modules/qscintilla-qt5/default.nix

```diff
@@ -5,47 +5,51 @@
 }:
 
 let
-  inherit (pythonPackages) buildPythonPackage isPy3k python sip_4 pyqt5;
+  inherit (pythonPackages) buildPythonPackage isPy3k python sip sipbuild pyqt5 pyqt-builder;
 in buildPythonPackage rec {
   pname = "qscintilla";
   version = qscintilla.version;
   src = qscintilla.src;
-  format = "other";
+  format = "pyproject";
 
   disabled = !isPy3k;
 
-  nativeBuildInputs = [ sip_4 qtbase ];
-  buildInputs = [ qscintilla ];
-  propagatedBuildInputs = [ pyqt5 ];
+  nativeBuildInputs = [ sip ];
```

Comment:
Would `pyqt5_sip` work better here?

Reply:
Is `pyqt5_sip` accessible outside of the pyqt recipe?

Reply:
Apologies for my slowness -- I did see it in the `propagatedBuildInputs`, but my nix skills may be a bit lacking as this is my first PR. I can't seem to get nixpkgs to recognize `pyqt5_sip` as an available package instead, whether it's including it instead of `sip`/`pyqt5` or trying it as an attribute of `pyqt5`. Regardless, the package seems to build as noted further down, but still happy to pursue this further.

Reply:
Was just a hint. Good work on this PR!